src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/ScoreVerificationTest.java

```diff
@@ -0,0 +1,82 @@
+package com.sap.sgs.phosphor.fosstars.model.score.oss;
+
+import static org.junit.Assert.fail;
+
+import com.sap.sgs.phosphor.fosstars.model.Feature;
+import com.sap.sgs.phosphor.fosstars.model.Parameter;
+import com.sap.sgs.phosphor.fosstars.model.Rating;
+import com.sap.sgs.phosphor.fosstars.model.RatingRepository;
+import com.sap.sgs.phosphor.fosstars.model.Score;
+import com.sap.sgs.phosphor.fosstars.model.Visitor;
+import com.sap.sgs.phosphor.fosstars.model.qa.ScoreVerification;
+import com.sap.sgs.phosphor.fosstars.model.qa.TestVectors;
+import com.sap.sgs.phosphor.fosstars.model.qa.VerificationFailedException;
+import com.sap.sgs.phosphor.fosstars.model.rating.oss.OssSecurityRating;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ScoreVerificationTest {
+
+  @Test
+  public void testVerification() {
+    ScoreCollector collector = new ScoreCollector();
+    RatingRepository.INSTANCE.rating(OssSecurityRating.class).accept(collector);
+
+    // add extra scores
+    collector.scores.add(new VulnerabilityLifetimeScore());
+
+    boolean failed = false;
+    for (Score score : collector.scores) {
+      String className = score.getClass().getSimpleName();
+      String testVectorsFileName = String.format("%sTestVectors.yml", className);
+      try (InputStream is = score.getClass().getResourceAsStream(testVectorsFileName)) {
+        TestVectors testVectors = TestVectors.loadFromYaml(is);
+        if (testVectors.isEmpty()) {
+          failed = true;
+          System.out.printf("No test vectors for %s%n", className);
```

Comment:
DO we really need these sysouts?

Reply:
Yes, those may be quite helpful when you add a new score and write a verification procedure for it.